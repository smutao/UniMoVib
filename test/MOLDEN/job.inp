This is a test job

 $contrl
   qcprog="molden"
   ifconc=.t.
   ISyTol=42
 $end

 $qcdata
   fchk="C6H6-VIBRATIONS-1.ref.mol"
 $end
