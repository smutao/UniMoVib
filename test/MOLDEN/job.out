 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      MOLDEN
 FCHK file:       C6H6-VIBRATIONS-1.ref.mol


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    C       6            0.00002223    1.39685741    0.00000000           12.00000000
     2    H       1           -0.00001217    2.48716206    0.00000000            1.00782504
     3    C       6           -1.21092367    0.69871049    0.00000000           12.00000000
     4    H       1           -2.15481615    1.24430733    0.00000000            1.00782504
     5    C       6           -1.21093055   -0.69866340    0.00000000           12.00000000
     6    H       1           -2.15478652   -1.24432214    0.00000000            1.00782504
     7    C       6           -0.00002170   -1.39687488    0.00000000           12.00000000
     8    H       1            0.00001217   -2.48718058    0.00000000            1.00782504
     9    C       6            1.21090938   -0.69870203    0.00000000           12.00000000
    10    H       1            2.15480134   -1.24429886    0.00000000            1.00782504
    11    C       6            1.21094589    0.69867186    0.00000000           12.00000000
    12    H       1            2.15480398    1.24433061    0.00000000            1.00782504
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   D6h 
 Molecular Point Group (Z+M) :   D6h 


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                             1         2         3         4         5         6         7         8         9        10
          Irreps           E2u       E2u       E2g       E2g       A2u       B1g       E1g       E1g       E2u       E2u
     Frequencies        400.30    401.56    596.10    602.92    661.19    706.25    833.57    833.76    949.50    949.98
 Force constants        0.2650    0.2663    1.2680    1.2894    0.2794    1.0820    0.5109    0.5113    0.7385    0.7397
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            11        12        13        14        15        16        17        18        19        20
          Irreps           B1g       A1g       B2u       E1u       E1u       B1u       E2g       E2g       A2g       B1u
     Frequencies        976.96    985.82    993.02   1029.98   1035.06   1143.90   1163.65   1165.97   1338.89   1347.66
 Force constants        0.6995    3.4519    3.7785    1.0571    1.0540    0.8050    0.8992    0.9041    1.3183    9.0849
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000

                            21        22        23        24        25        26        27        28        29        30
          Irreps           E1u       E1u       E2g       E2g       B2u       E2g       E2g       E1u       E1u       A1g
     Frequencies       1466.71   1471.80   1593.03   1594.12   3087.28   3096.78   3097.76   3111.83   3112.82   3121.74
 Force constants        2.5788    2.6301    8.3405    8.3659    6.0846    6.1430    6.1462    6.2458    6.2513    6.3113
  IR intensities        0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000    0.0000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        78.046950    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :        12
 The D6h  point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                317.725317          318.191355          635.916672
           X                        -1.000000            0.000125            0.000000
           Y                         0.000125            1.000000            0.000000
           Z                         0.000000            0.000000            1.000000

 Rotational temperatures             0.272606            0.272207            0.136203    Kelvin
 Rot. constants A, B, C              0.189471            0.189193            0.094666    cm**-1
                                     5.680193            5.671874            2.838015    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.097665           61.285586
 Thermal correction to Energy               :            0.102180           64.119012
 Thermal correction to Enthalpy             :            0.103124           64.711497
 Thermal correction to Gibbs Free Energy    :            0.072492           45.489171


 Job terminated correctly! Tue May 02 17:48:54 2017
